microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,93 @@
+
+The purpose of this article is to provide the following information.  
+
+*   The current approach that the WebView2 team uses for browser features.  
+*   The status of features and the reasoning behind the state.  
+*   The ability for features to be turned on \(or turned off\).  
+
+## Guiding Principles  
+
+Browser features within the context of WebView2 should adheres to the following guidelines for most WebView2 features.  
+
+*   Most of the WebView2 features work the same as Microsoft Edge features.  The following exceptions define why a feature is turned off.  
+    *   It doesn't make sense within the context of WebView2.  
+    *   It has an extended service cost.  
+*   The WebView2 features shouldn't include any Microsoft Edge branding.  
+    
+## Features  
+
+The following list displays the WebView2 features that differ from the new Microsoft Edge browser.  
+
+About this table:
+- Feature: The name of the feature.
+- Default: The default experience on a new WebView2 instance
+- Configurable: If the feature can be toggled on/off via our API surface or command line switch.
+- Details: 
+
+| Feature | Default | Configurable | Details |  
+|:--- |:--- |:--- | :--- | 
+| Autofill for Addresses | Enabled | Yes | This feature is turned on by default, you may turn it on or off via WebView2 Autofill API. |  
+| Autofill for Passwords | Enabled | Yes | This feature is turned on by default, you may turn it on or off via WebView2 Autofill API. |  
+| Autofill for Payments | Disabled | No | This feature is turned off. | 
+| Browser Extensions | Disabled | No | This feature is turned off. |   
+| Browser Task Manager | Disabled | No | This feature is turned off. |  
+| Continue-where-I-left-off prompt | Disabled | No | This feature is turned off. |  
+| Downloads | Enabled | Yes | WebView2 provides an api that allows you to customize the download UI to manipulate downloads, such as block, redirect, save, pause, and so on.  For more information, navigate to <!--[download api][Webview2ReferenceDownloadApi]-->. |  
+| Family Safety | Disabled | No | This feature is turned off. |  
+| Hotkeys | See Details | See Details | The following shortcuts are turned off by default:  `Ctrl`+`F` and `Ctrl`+`S`. The keys that are default disabled are generally keys that either don't make sense for WebView or simply cause problems of some sort or other, and are not toggle-able. Alternatively, ou can listen for a key combination in AcceleratorKeyPressed and create a custom response if needed.|  
+| IE Mode | Disabled | No | This feature is turned off. |  
+| Immersive Reader | Disabled | No | This feature is dependent on the browser UI for control and triggering.  This feature is turned off. |  
+| Intrusive Ads | Enabled* | No | The UI for this feature is removed, however the underlying functionality persists. |  
```

Comment:
Are we sure this is enabled? If this is related to the abusive ads service, this is turned off.

Reply:
@liminzhu based on Josh's notes and this [scenario](https://microsoft.visualstudio.com/Edge/_workitems/edit/27563881) I believe this is enabled in WebView2. Is abusive ads different from intrusive ads?

Reply:
Yah we turned off that service in Edge 89, assuming abusive and intrusive ads are the same thing, I wouldn't be surprised that it worked when Josh looked at it. Can you check with Vincent Wehren? He should be able to tell you whether they are the same thing and if they are, then it's turned off in WebView2.

Reply:
I'm like 90% sure they are the same :).

Reply:
Vincent confirmed they are the same, changing this to disabled. thanks!